.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Can we retain 20.04 too?

Reply:
I don't see a lot of reasons why. These two versions both have cgroup v1, there are some difference in kernel and tools version but not much.

Reply:
Ubuntu 22.04 uses cgroup v2

Reply:
> Ubuntu 22.04 uses cgroup v2

Wow, I missed that. In this case we definitely need 20.04, too.